conf/pf.conf.defaults

```diff
@@ -1069,6 +1069,11 @@ timing_stats_level=5
 # Set the max size of DHCP packets
 # Do not change unless you know what you are doing
 pfdhcplistener_packet_size=576
+#
+# advanced.source_to_send_sms_when_creating_users
+#
+# The source to use to send an SMS when creating a user
+source_to_send_sms_when_creating_users=
```

Comment:
No as this is an advanced feature.
Most people will not know or understand why they would have to select a user's carrier in order to send an SMS to that user.